code/danielle/Any_API/any_api.js

```diff
@@ -0,0 +1,29 @@
+const btn = document.getElementById('btn')
+const numberTrivia = document.getElementById('numberInput')
+let banner = document.getElementById('banner')
+let qualityBanner = document.getElementById('quality')
+
+btn.addEventListener('click', function () {
+    const number = numberTrivia.value
+    if (number == '') {
+        return
+    }
+    getData(number)
+
+    })
+
+    function getData(number) {
+        fetch(`http://numbersapi.com/${number}/trivia`) // ${}
+            .then(function (response) {
+                console.log(response)
+                return response.text()
+            })
+            .then(function (data) {
+                console.log(data)
```

Comment:
@dlcantu this isn't outputting data on a page. Make sure to implement that part!

Reply:
It works now!